e2e/test/scenarios/dashboard/reproductions/35037-recently-viewed-link-editing-dashboard-link-card.cy.spec.js

```diff
@@ -0,0 +1,69 @@
+import {
+  createLinkCard,
+  editDashboard,
+  restore,
+  visitDashboard,
+} from "e2e/support/helpers";
+import { ORDERS_DASHBOARD_ID } from "e2e/support/cypress_sample_instance_data";
+
+const TEST_DASHBOARD_NAME = "Orders in a dashboard";
+const TEST_QUESTION_NAME = "Question#35037";
+
+describe("should not redirect users to other pages when linking an entity (metabase#35037)", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    cy.intercept("GET", "/api/search?q=*").as("search");
+    cy.intercept("GET", "/api/activity/recent_views").as("recentViews");
+  });
+
+  it("should not redirect users to recent item", () => {
+    visitDashboard(ORDERS_DASHBOARD_ID);
+    // get the URL of page and store it in a variable
+    const initialUrl = cy.url();
```

Comment:
That comment brings no value.

```suggestion
    const initialUrl = cy.url();
```